test/lint/test_runner/src/main.rs

```diff
@@ -180,6 +185,31 @@ fn lint_subtree() -> LintResult {
     }
 }
 
+fn lint_py_mut_arg_default() -> LintResult {
+    let bin_name = "ruff";
+    let checks = ["B006", "B008"]
+        .iter()
+        .map(|c| format!("--select={}", c))
+        .collect::<Vec<_>>();
+    let files = check_output(git().args(["ls-files", "--", "*.py"]))?;
```

Comment:
```suggestion
    let files = check_output(
        git()
            .args(["ls-files", "--", "*.py"])
            .args(get_pathspecs_exclude_subtrees())
    )?;
```
nit: exclude subtrees from this check